Source/WebCore/platform/graphics/Color.h

```diff
@@ -295,9 +295,10 @@ class Color {
 
 inline void add(Hasher& hasher, const Color& color)
 {
-    if (color.isOutOfLine())
-        add(hasher, color.asOutOfLine().unresolvedComponents(), color.colorSpace(), color.flags());
-    else
+    if (color.isOutOfLine()) {
+        // FIXME: We should not need SUPPRESS_UNCOUNTED_ARG as unresolvedComponents() is trivial and inline.
```

Comment:
Can we file a radar and put that here? That'll allow us to find these quickly when the fix lands.